src/sonic-config-engine/minigraph.py

```diff
@@ -696,46 +697,40 @@ def parse_cpg(cpg, hname, local_devices=[]):
                     keepalive = 60
                 nhopself = 1 if session.find(str(QName(ns, "NextHopSelf"))) is not None else 0
 
+                # choose the right table and admin_status for the peer
+                voq_chassis = session.find(str(QName(ns, "VoQChassisInternal")))
+                if voq_chassis is not None and voq_chassis.text == "true":
+                    table = bgp_voq_chassis_sessions
+                    admin_status = None
```

Comment:
should the internal sessions be up by default, if not the admin status will be changed by CLI ?

Reply:
It was agreed the sessions would be up by default so that routing works as intended as soon as any eBGP session comes up. The admin status should be changeable with the CLI though this PR does not cover the CLI. Are you saying admin_status should be 'up', rather than not-specified? Not-specified is not 'down' so the sessions would be configured up either way.

Reply:
My point was should we set the admin_status to up here as we want these session to up by default.

Reply:
Thanks. I have changed it to 'up'. Please take a look!